addons/website_crm_partner_assign/static/src/interactions/crm_partner_assign.js

```diff
@@ -0,0 +1,153 @@
+import { Interaction } from "@web/public/interaction";
```

Comment:
No need to adapt manifest ?

Reply:
No, it already import every `.js` file in `src`